dashboard/UI-Test/test.yaml

```diff
@@ -0,0 +1,65 @@
+sequence:
+- testcase 1
+- testcase 2
+test:
+- testcase 1:
+  - scenario: invalid pbench-dashboard login
+  - step 1:
+      action: goto
+      target: http://10.1.170.201
+      targets:
+      - http://10.1.170.201
+  - step 2:
+      action: click
+      target: Login
+      targets:
+      - Login
+  - step 3:
+      action: input
+      target: //*[@id="username"]
+      value: user
+      targets:
+      - //*[@id="username"]
+  - step 4:
+      action: input
+      target: //*[@id="password"]
+      value: password
+      targets:
+      - //*[@id="password"]
+  - step 5:
+      action: click
+      target: Login
+      wait after action: 3
+      assert: text=Bad login
+      targets:
+      - Login
+- testcase 2:
+  - scenario: Valid pbench-dashboard login
+  - step 1:
+      action: goto
+      target: http://10.1.170.201
+      targets:
+      - http://10.1.170.201
+  - step 2:
+      action: click
+      target: Login
+      targets:
+      - Login
+  - step 3:
+      action: input
+      target: //*[@id="username"]
+      value: vishalvvr
```

Comment:
We don't want to hardcode user and password, either; and certainly not to your account! While we could stand up a CI pipeline pod with a pre-configured test user, it'd be much better to have this imported from outside via an environment variable or similar.